eppoclient/configurationstore.go

```diff
@@ -31,12 +31,6 @@ func (c *configuration) refreshBanditFlagAssociations() {
 	c.banditFlagAssociations = associations
 }
 
-// Return `true` if `flagKey` has associated bandits.
```

Comment:
No longer needed in code (🪓  to appease golint)

Reply:
🪓